BasicSample/Assets/Shared/Scripts/ProjectConfiguratorWindow.cs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.OpenXR.Remoting;
+using System;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEditor.XR.Management;
+using UnityEditor.XR.OpenXR.Features;
+using UnityEngine;
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features;
+
+
+namespace Microsoft.MixedReality.OpenXR.BasicSample
+{
+    enum ConfigurationSelection
+    {
+        None,
+        RunNativelyonHL2,
+        RunNativelyonPCVR,
+        RunRemotelyonUWP,
+        RunRemotelyonWin32
+    }
+    public class ProjectConfiguratorWindow : EditorWindow
+    {
+        private ConfigurationSelection selectedConfiguration, previousSelectedConfiguration;
+        private const float Default_Window_Height = 300.0f;
+        private const float Default_Window_Width = 300.0f;
+
+        public static ProjectConfiguratorWindow Instance { get; private set; }
+        public static bool IsOpen => Instance != null;
+
+        [MenuItem("ProjectConfigurator/Quick Setup", false, 499)]
+        private static void ShowWindowFromMenu()
+        {
+            ShowWindow();
+        }
+
+        public static void ShowWindow()
+        {
+            // There should be only one configurator window open as a "pop-up". If already open, then just force focus on our instance
+            if (IsOpen)
+            {
+                Instance.Focus();
+            }
+            else
+            {
+                var window = CreateInstance<ProjectConfiguratorWindow>();
+                window.titleContent = new GUIContent("Basic Sample Project Quick Configurator", EditorGUIUtility.IconContent("_Popup").image);
+                window.position = new Rect(Screen.width / 2.0f, Screen.height / 2.0f, Default_Window_Height, Default_Window_Width);
+                window.ShowUtility();
+            }
+        }
+
+        private void OnGUI()
+        {
+            EditorGUIUtility.labelWidth = 250;
+            bool remoting = false;
+            BuildTargetGroup targetGroup;
+            previousSelectedConfiguration = selectedConfiguration;
+            selectedConfiguration = (ConfigurationSelection)EditorGUILayout.EnumPopup("Select one of the following configurations:", selectedConfiguration);
```

Comment:
I don't think cancelling is a viable option here unfortunately, I did try to check if this can work, but the moment user hits "apply", the cursor is spinning and won't let user click anything until applying finishes.

Reply:
the "cancel" meant for the user to close the dialog without applying the changes in the combobox.

Reply:
maybe close the dialog is enough, no need for the "cancel" button.   the screenshot above is helpful.

Reply:
unfortunately as I told, this did not work